code/parse/sexp.cpp

```diff
@@ -14668,16 +14649,9 @@ void sexp_ingame_ship_kamikaze(ship *shipp, int kdamage)
 
 	ai_info *aip = &Ai_info[shipp->ai_index];
 
-	if (kdamage > 0)
-	{
-		aip->ai_flags |= AIF_KAMIKAZE;
-		aip->kamikaze_damage = kdamage;
-	}
-	else
-	{
-		aip->ai_flags &= ~AIF_KAMIKAZE;
-		aip->kamikaze_damage = 0;
-	}
+	aip->ai_flags.set(AI::AI_Flags::Kamikaze, kdamage > 0);
+	aip->kamikaze_damage = kdamage;
```

Comment:
This might not behave like the original code since that assigned 0 if the value is `<= 0` but the new version uses the parameter value directly. If the function is never called with a value `< 0` then that should be enforced by an assertion.